src/types.js

```diff
@@ -137,12 +137,15 @@ export type CollectionsListOptions = {
   userId?: string
 };
 
-export type AccessToken = ?string | ShareDescriptor;
+export type AccessToken = ?string;
 export type AccessTokenOption =
   | AccessToken // TODO: Deprecate?
   | (() => AccessToken) // TODO: Deprecate
   | (() => Promise<AccessToken>);
 
+export type ShareIdentifier = string | ShareDescriptor;
```

Comment:
nit: 

```suggestion
type ShareIdentifier = string | ShareDescriptor;
```

or just inline `string | ShareDescriptor` below. We should be selective about which types we introduce to the public api

Reply:
Went ahead and inlined it. I'll also make sure to update this after Enrique's PR lands.